boards: nxp: Add missing RW board feature tables

decsny opened the pull request:
Add missing items to RW board feature tables.

Commit:
boards: nxp: Add missing RW board feature tables

Add missing items to RW board feature tables.

Signed-off-by: Declan Snyder <declan.snyder@nxp.com>

danieldegrasse commented on boards/nxp/frdm_rw612/doc/index.rst:
Should we add `- netif:eth` to the supported table for this board? It doesn't seem to be present

Commit:
boards: frdm_rw612: Add ethernet to twister yaml

Add netif:eth to twister yaml for frdm_rw612

Exclude from wifi test because of binary blob requirement

Signed-off-by: Declan Snyder <declan.snyder@nxp.com>

danieldegrasse commented:
We need a fix like #78415 for the FRDM board as well it seems. Sorry, should have realized this when I made my earlier suggestion
